moler/cmd/commandtextualgeneric.py

```diff
@@ -177,13 +179,15 @@ def has_endline_char(self, line):
             return True
         return False
 
-    def data_received(self, data):
+    def data_received(self, data, timestamp):
```

Comment:
maybe `recv_time` naming instead of `timestamp` (which suggest textual format in logs - logs timestamp) ?

Moreover, `recv_time` is in accordance with your description: `with the moment when the data was read from connection`

Reply:
ok. recv_time